### PR TITLE
Removed old tabbar icons and added new tabbar icons.

### PR DESCRIPTION
**Pull request by roberthein**
Removed old tabbar icons and added new tabbar icons.

img_4230

### CODE REVIEW

File: Toshi/Controllers/Favorites/FavoritesNavigationController.swift [hunk 1]
```diff
@@ -30,8 +30,7 @@ public class FavoritesNavigationController: UINavigationController {
     public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
         super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
 
-        tabBarItem = UITabBarItem(title: "Favorites", image: #imageLiteral(resourceName: "favourites"), tag: 1)
-        tabBarItem.selectedImage = #imageLiteral(resourceName: "favourites-selected")
+        tabBarItem = UITabBarItem(title: "Favorites", image: #imageLiteral(resourceName: "tab4"), tag: 1)
```
Comment (yuliaveres) [inline Toshi/Controllers/Favorites/FavoritesNavigationController.swift]: localized ? 👿

File: Toshi/Controllers/Messaging/ChatsNavigationController.swift [hunk 2]
```diff
@@ -30,8 +30,7 @@ public class ChatsNavigationController: UINavigationController {
     public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
         super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
 
-        tabBarItem = UITabBarItem(title: "Recent", image: #imageLiteral(resourceName: "chats"), tag: 0)
-        tabBarItem.selectedImage = #imageLiteral(resourceName: "chats-selected")
+        tabBarItem = UITabBarItem(title: "Recent", image: #imageLiteral(resourceName: "tab2"), tag: 0)
```
Comment (yuliaveres) [inline Toshi/Controllers/Messaging/ChatsNavigationController.swift]: localized ? 👿

File: Toshi/Controllers/Settings/SettingsNavigationController.swift [hunk 3]
```diff
@@ -28,8 +28,7 @@ public class SettingsNavigationController: UINavigationController {
     public override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: Bundle?) {
         super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
 
-        tabBarItem = UITabBarItem(title: "Me", image: #imageLiteral(resourceName: "me"), tag: 0)
-        tabBarItem.selectedImage = #imageLiteral(resourceName: "me-selected")
+        tabBarItem = UITabBarItem(title: "Me", image: #imageLiteral(resourceName: "tab5"), tag: 0)
```
Comment (yuliaveres) [inline Toshi/Controllers/Settings/SettingsNavigationController.swift]: localized ?

File: Toshi/Controllers/TabBarController.swift [hunk 4]
```diff
@@ -52,9 +52,9 @@ open class TabBarController: UITabBarController {
 
     internal lazy var placeholderScannerController: UIViewController = {
         let controller = UIViewController()
-        controller.tabBarItem = UITabBarItem(title: "Scan", image: #imageLiteral(resourceName: "scan"), tag: 0)
+        controller.tabBarItem = UITabBarItem(title: "Scan", image: #imageLiteral(resourceName: "tab3"), tag: 0)
```
Comment (yuliaveres) [inline Toshi/Controllers/TabBarController.swift]: and localized

File: Toshi/Controllers/Favorites/FavoritesNavigationController.swift (hunk 1 above)
Comment (roberthein) [reply Toshi/Controllers/Favorites/FavoritesNavigationController.swift]: done

File: Toshi/Controllers/Messaging/ChatsNavigationController.swift (hunk 2 above)
Comment (roberthein) [reply Toshi/Controllers/Messaging/ChatsNavigationController.swift]: done

File: Toshi/Controllers/Settings/SettingsNavigationController.swift (hunk 3 above)
Comment (roberthein) [reply Toshi/Controllers/Settings/SettingsNavigationController.swift]: done

File: Toshi/Controllers/TabBarController.swift (hunk 4 above)
Comment (roberthein) [reply Toshi/Controllers/TabBarController.swift]: done

**Pull request merged**